<?xml version="1.0"?>
<launch>
   <arg name="ontology" default="$(find erl2)/cluedo_ontology.owl" />

   <node name="ontology_interface"  pkg="erl2" type="ontology_interface.py" output="screen">
      <param name="ontology"               value="$(arg ontology)"/>
   </node>
   <node name="go_to_point"  pkg="erl2" type="go_to_point.py"/> 
   <node name="simulation" pkg ="erl2" type="simulation"  /> 
    
</launch>
